cli/ballerina-cli/src/main/resources/cli-help/ballerina-test.help

```diff
@@ -125,3 +130,7 @@ EXAMPLES
        Run the test functions with the code coverage and generate a coverage
        report in the XML format.
            $ bal test --test-report --code-coverage --coverage-format=xml
+
+       Run the tests with the code coverage and exclude files from coverage
+       calculation.
+           $ bal test --code-coverage --excludes=<./Relative src path>/<Absolute src path>
```

Comment:
As we've mentioned that path can be relative or absolute, shall we remove that information from here?
wbt `--excludes=<source_path1>,<source_path2>` instead?